cypress/e2e/patient_spec/encounter-notes.cy.ts

```
@@ -0,0 +1,78 @@
+  beforeEach(() => {
+    cy.loginByApi("devdoctor"); // logging in as doctor
+    cy.visit("/");
+    facilityCreation.selectFacility("GHC Trikaripur");
+  });
+
+  it("create multiple threads and add messages", () => {
+    patientEncounter
+      .navigateToEncounters()
+      .openFirstEncounterDetails()
+      .openNotesTab()
+      
```

ESLint

[error] Replace `⏎······` with `;`

(prettier/prettier)